project_status/tests/test_status.py

```
@@ -0,0 +1,30 @@
+class TestProjectStatus(SavepointCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.ProjectStatus = cls.env["project.status"]
+        cls.status = cls.ProjectStatus.create(
+            {
+                "name": "New Status",
+            }
+        )
+
+    def _create_project(self):
```

Ruff (0.11.9)

Missing return type annotation for private function `_create_project`

(ANN202)